docs/packaging/your-first-package.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Your First Package
+summary: Your First Package
+---
+
+# Your First Package
+
+This page will guide you through the steps required to build your first package. This is really a test to make sure that your [build environment is set up correctly](docs/packaging/index.md). You will clone an existing package, build it, then install it, and finally remove it.
+
+## Cloning and building `nano`
+
+For this example we will use the text editor `nano`.
+
+Move to your build directory. Your directory may have a different name or location.
+
+```bash
+cd ~/solus-builds
+```
+
+Clone the `nano` repository, then move into it.
+
+```bash
+make nano.clone
+cd nano
+```
+
+Increase the release number by one ("bump" the package) and then build the package.
+
+```bash
+make bump
+make
+```
+
+You will be prompted to enter your password by sudo and the `solbuild` tool will build the package.  
+If the build is successful, you will have a `nano` package file with a name like `nano-7.2-161-1-x86_64.eopkg`. Run the `ls` tool to check:
```

Comment:
I wonder if we should mention the practice of using `git diff` after `make bump` to make sure that the results are as expected; sometimes `make bump` does weird things with newlines in other parts of the file.